### PR TITLE
Import css screens to react

### PR DESCRIPTION
**Pull request by Abhijeet43**
- No functionality has been added 
- Static screens for e-commerce.

Added:
screens for login, signup, cart management, wish list, homepage and single product page.
CSS files for respective pages and components.
Images for homepage screen and a product image.

Homepage:
home

Cart:
cart

Wishlist:
Wishlist

ProductListing:
ProductListing

ProductDetails:
ProductDetails

Login:
Login

SignUp:
Signup

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
1.all the screens looks great
2.the folders are well organised
3.nice color palette for ui

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          <li className="footer-item">
+            <a className="footer-link" href="#">Settings</a>
+          </li>
+        </ul>
+      </div>
+    </footer>
+  )
+}
+
+export { Footer }
```
Comment (VishalPatil18) [inline src/components/Footer/Footer.jsx:46]: You could add an empty line at the end of the file to avoid such warnings by Github

**Review by AtulPant2704 — APPROVED** [review]
- Very well formatted code with proper indentation
- Having index.js files for exports in every folder made imports really easy

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      </div>
+      <div className="footer-nav">
+        <ul>
+          <li className="footer-item">
+            <a className="footer-link" href="../cart/cart.html">My Cart</a>
+          </li>
+          <li className="footer-item">
+            <a className="footer-link" href="../wishlist/wishlist.html"
+              >My Wishlist</a
+            >
```
Comment (VishalPatil18) [inline src/components/Footer/Footer.jsx:35]: Indentation could be improved

File: src/components/ProductCard/ProductCard.jsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import "./ProductCard.css";
+
+const ProductCard = ({
+  cardImage,
+  cardTitle,
+  cardBadge,
+  cardPrice,
+  cardPriceBefore,
+  cardDiscount,
```
Comment (VishalPatil18) [inline src/components/ProductCard/ProductCard.jsx:10]: As we know that this is the data for a card then we could avoid prefixing `card` before all the props and could simply write `image`, `title`, etc.

**Review by VishalPatil18 — APPROVED** [review]
Just a few suggestions were added, the rest looks quite good. Great work @Abhijeet43 👍

File: src/components/ProductCard/ProductCard.jsx (hunk 3 above)
Comment (Abhijeet43) [reply src/components/ProductCard/ProductCard.jsx:10]: Thanks Vishal for the reveiw, Will look into the suggestions...

**Pull request merged**